tf2_eigen/package.xml

```diff
@@ -8,13 +8,13 @@
   <license>BSD</license>
 
   <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>cmake_modules</build_depend>
```

Comment:
This needs `<build_export_depend>` too since it's in the header and will be needed downstream.